testsuite/arquillian-utils/pom.xml

```diff
@@ -137,6 +137,23 @@
         </dependency>
     </dependencies>
 
+    <profiles>
```

Comment:
I'm not sure this needed. This module contains no tests. The dependency should be added to the modules that use it instead of this module where it would be picked up transitively.

Reply:
Tnks, removed